packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -45,6 +45,9 @@ module.exports = (resolve, rootDir) => {
     moduleNameMapper: {
       '^react-native$': 'react-native-web',
     },
+    globals: {
+      __TS_CONFIG__: resolve('template/tsconfig.json'),
```

Comment:
This won't be inside `template` once an app has been initialised. This was used previously:

```
const tsconfigPath = require('app-root-path').resolve('/tsconfig.json');
```

which worked well!